### PR TITLE
Initial stab at TWBX support

### PR DESCRIPTION
**Pull request by t8y8**
- Created a few utility functions for working with Tableau ZIP Files. Not sure where they should go long term, they're hanging out in the workbook module for now
- Save and Save As are supported. There is probably a little more refactoring that can be done to remove duplication, but this is much better than the prototype :)
- Tests added to verify actual saving and unpacking.

### CODE REVIEW

File: tableaudocumentapi/workbook.py [hunk 1]
```diff
@@ -3,10 +3,57 @@
 # Workbook - A class for writing Tableau workbook files
```
Comment (LGraber) [inline tableaudocumentapi/workbook.py]: Lets talk at work. A twbx is much more than just the workbook and needs its own representation. With just the twb ... it is not so interesting or useful. While I admit that we only have the ability to edit the xml for twb right now, you could see people using this api to potentially one day create an extract using our Extract API, create a workbook and then package them up together. In this example, the packaging also affects the path specified in the connection string itself. Lets spend a bit more time talking about this. Thanks for getting it started though!!!

**Comment by t8y8** [comment]
I've updated the PR against master, applies clean and passes tests.

@RussTheAerialist this is the PR I mentioned. I think it's still incomplete (I need to add tdsx support) and am open to better ideas on where to place the zip-handling code.

File: test.py [hunk 2]
```diff
@@ -17,6 +19,8 @@
...
...ySO8k7ilSsJKwl2w+jbhZ5bPmUP+/VNuKso9LcXza8jLpXvBOdk9kW/pzgq7/mPd18/PO/4CnlSp/PzwTHtqLyRhV9C/ohjpO/78F6qYb6jKxxjzw7sqNlZaE+/9j/Fp398jaZvsz9BQcesF/LN/ZhBhvF3G35+9NOF/tb5H3PvXfz7Ff679zZGLVthXAne2HAd44jN8LRIHYIzPzrymMgVu1hOr8z95DgIzXU73O7CWR+93t5Lfx4nr46f95DnQOy8ns7z6IfXwGhPR873kLOUniSuBE+HPeQ8AM3vL4X7LXyuR/uu7Zz7cfOr/7SvaKItx8vv6b+btr8P6gC8+3aEA+Z82f+gGCfb76G/v9XrX8j9R/f0Sv88KnR7FJUZ0p1Ez/bk5QxcBY19ZZ+19n0N+u2zMYXjj+B4Izi+CI4n1L+mvC7KmfOQle4vT/0Zcr9w2NNF2N/L2aLuXHkeM/qvQ+9/h+gfHCcExwXBcUDwuq/+9vsr8Tl7dAP0++v/4vV+G3bM21VY//10vY8x/UX99mn2yycj5Iw80rcZwe+emTbjtk/8etvnC/2r7au3tG3O6wf87XM35Qx9uLC8Hs/nvt4b6FewQnwf/d0V6oyyuycOMcj4rwI7ng6xx4yPzNxJ/gijp9nDLq4apfR6xXJ616g95DgAzRyMd40NmzmcR4+0Bkbl8MpwOBfHQDOvk19G2KxhG6m3xsql12w939fQPnsqy9NwB63rYVjov6hRThn1NxEWy+mVuZ8cB6G29kOHdP7YzJP5v0fn3d84zv7rJVI2XQdj7qX9aVdIeN7NJoXjLL7V0lw/9X5LYm+zR++CpjT/vVVeaTyp0qdSpjR/WlPa7rutPJ1JPUWCXi3nSWM8tJdWa67jXkojy3nE/HOcfz5q6fnG7Bn/mLGBaXne3UGt3H+aVbRT5E7U/ga8I28fsFsTNHrnf1fAcfaYg7JVmlNONjOrYb8/IDtfMEccZ8Rjlv8LUEsBAgAAFAAGAAgAV1S4SMbSoj9UDAAACzwAAAkAAAAAAAAAAQAAAAAAAAAAAEJvb2sxLnR3YlBLAQIAABQABgAIAFdUuEjiq04zHiIAAJq+AAAgAAAAAAAAAAAAAAAAAHsMAABEYXRhL0RhdGFzb3VyY2VzL3h5IChUZXN0VjEpLnRkZVBLBQYAAAAAAgACAIUAAADXLgAAAAA='
```
Comment (RussTheAerialist) [inline test.py]: Can we just move this to be an external file rather than continuing to encode it in the test file?

**Comment by RussTheAerialist** [comment]
Can you resubmit this against the newly created `development` branch so that new features are not immediately added to master and master represents what is available on pypi (coming soon)

File: test.py (hunk 2 above)
Comment (t8y8) [inline test.py]: Yeah, I may as well do that for all the tests while I'm there -- got a preference for how that folder structure should look? When I rebase it'll pick up your changes to the test module... maybe tests/assets/*.twb ?

Comment (RussTheAerialist) [inline test.py]: I like that, so let's go with that.

**Comment by RussTheAerialist** [comment]
Closing this PR because it's superseded by #43

**Pull request closed without merging**